static/js/lightbox.js

```diff
@@ -50,6 +52,11 @@ export class PanZoomControl {
             }, 0);
         });
 
+        this.panzoom.on("zoom", (e) => {
```

Comment:
Should this maybe be ["zoomend"](https://github.com/anvaka/panzoom#events-notification)? We're using "panend" instead of "pan" to handle panning.

Reply:
The zoomend event doesn't fire when using the touchpad or mousewheel, only when called directly via js 😭. https://github.com/anvaka/panzoom/issues/250 Definitely a fix that should be submitted upstream when there is time.

Reply:
Added a link to one of the upstream issues.